_posts/2022-07-18-kubeflow-1.5.1-release.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Kubeflow 1.5.1 Release Announcement"
+layout: post
+toc: true
+comments: true
+image: images/logo.png
+hide: false
+categories: [release]
+permalink: /kubeflow-1.5-release/
+author: "Kubeflow 1.5 Release Team"
+---
+
+The Kubeflow Community is excited to announce the release of [Kubeflow 1.5.1](https://github.com/kubeflow/manifests/releases/tag/v1.5.1). The Kubeflow 1.5.1 release provides a valuable enhancement to the caching feature of Kubeflow Pipeline (KFP) for any Kubernetes distribution with strict security policies for user application certificates. Caching is one of the crucial features of KFP which enables skipping a step that has already been executed which saves time and money. This release enables the caching feature to be used on distributions like AWS, Mercedes Benz cluster etc.
+
+Kubeflow 1.5.1 includes the enhancements for the Kubeflow Pipelines cache server and the cert generation mechanism [#2165](https://github.com/kubeflow/manifests/issues/2165). These enhancements improve the operations of Kubernetes clusters with strict security policies, especially in clusters where non-Kubelet applications cannot create API server type certificates.
```

Comment:
Do we want to call out that we improve the operations of kubernetes clusters with strict security policies by relying on cert-manager?

Reply:
Good question.   Do you have an alternative explanation or definition for the 1.5.1 functionality that we can use?  If not, could we please consider /lgtm ?   We have several projects coming up.

Reply:
I think it's fine the way it is. Especially the last sentence, regarding API server type certificates completely explains why someone should care about this patch release and who the affected users are.

IMHO it should be OK the way it currently is

Reply:
Just want to make sure my comment is correctly describing what I mean:

I suggest the following content: 
```
These enhancements improve the operations of Kubernetes clusters with strict security policies, especially in clusters where non-Kubelet applications cannot create API server type certificates. Kubeflow Pipelines introduced dependency on cert-manager for this enhancement.
```

Please let me know if this makes sense to you.

Reply:
Ah now I see. Sure, this makes it even more concrete!

Reply:
Added the proposed changes